### ISSUE TITLE
Issue with the paf file

### ISSUE THREAD

**Issue by anselal** [garethflowers/vscode-portable#53]
Hi there,

Minor issue here.
I downloaded the portable version from sourceforge and when I opened the file I got the following error.

![vscode_portable](https://user-images.githubusercontent.com/8732668/47793257-8d110a00-dd26-11e8-8b6e-a7f82b638d9c.png)

Is seems that the file has to be renamed to `VSCodePortable_Latest_online.paf.exe` in order to be able to download files from the internet. As it indicates this is so the user knows that the file will download something from the network.

**Comment by RoyiAvital** [comment]
The online of the latest version must be updated with the new options of VS Code (Portable flags).
This is based on a version from 2 months ago (1.26).

**Comment by RoyiAvital** [comment]
@garethflowers , Any chance you update the Online Installer with the latest flags enabled on VS Code?

The online installer was created prior to the time they were enabled.

**Comment by crazy-max** [comment]
@RoyiAvital You can check this portable alternative if you want : https://github.com/portapps/vscode-portable

**Comment by RoyiAvital** [comment]
@crazy-max ,
Thank you for the offer.
I don't see that you use the flags of VS Code bu you alter Environment Variables.

By the way, it would be great if you added MiKTeX + TeXStudio + TeXMaker to the list of your applications.

**Comment by RoyiAvital** [comment]
They have a flag to set the directory without any System Environment Variable.

That's what I meant.

**Comment by garethflowers** [comment]
I'm not sure which flags you are referring to that have changed?

We currently rely on `user-data-dir` and `extensions-dir` command line arguments to set the relevant data directories.

**Comment by RoyiAvital** [comment]
@garethflowers ,

Could you update the `Visual Studio Code Portable - Latest` release to also use those flags?

**Comment by garethflowers** [comment]
New version of [Visual Studio Code Portable - Latest v1.0.1](https://github.com/garethflowers/vscode-portable/releases/tag/latest-v1.0.1) has been released.

**Comment by anselal** [comment]
Is the version number correct ? It doesnt match the version of vscode

**Comment by RoyiAvital** [comment]
> New version of [Visual Studio Code Portable - Latest v1.0.1](https://github.com/garethflowers/vscode-portable/releases/tag/latest-v1.0.1) has been released.

This is perfect!
By the way, any chance for 64 Bit version?

**Comment by anselal** [comment]
Cool !!!

**Comment by anselal** [comment]
I got confused because it went from [v1.26.0](https://github.com/garethflowers/vscode-portable/tree/v1.26.0) to [v1.0.0](https://github.com/garethflowers/vscode-portable/tree/v1.0.0) to [v1.0.1](https://github.com/garethflowers/vscode-portable/tree/v1.0.1)

**Comment by garethflowers** [comment]
Yeah, sorry - didn't know how best to handle the versioning. I'm wondering if I should fork the repo just for the "latest" versions.

**Comment by weklost** [comment]
@crazy-max seems to have a better handling on versioning (latest is 1.30.2 on his repo) : https://github.com/portapps/vscode-portable/releases

**Comment by RoyiAvital** [comment]
> Yeah, sorry - didn't know how best to handle the versioning. I'm wondering if I should fork the repo just for the "latest" versions.

@garethflowers ,
Do you intend to start releasing dedicated releases?
It seems it will take too much of your time for little gain.

I'd say stick with this repository and just do the latest version.

**Comment by garethflowers** [comment]
I'd rather not keep doing dedicated releases - they are a pain to keep up with.

:+1: for 64/32 bit versions.
